masonry/src/widgets/flex.rs

```diff
@@ -294,12 +294,12 @@ impl Flex {
         self
     }
 
-    #[expect(missing_docs, reason = "TODO")]
+    /// Returns the number of children (widgets and spacers) this flex container has.
     pub fn len(&self) -> usize {
         self.children.len()
     }
 
-    #[expect(missing_docs, reason = "TODO")]
+    /// Returns true if this flex container has no children (widgets or spacers).
```

Comment:
We used to be consistent in having `true` wrapped in backticks. We should get back to that consistency.